### PR TITLE
Update Docs for Local Camera Switcher

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What
Update story book docs about local camera switcher doc. Introduce Storybook functionality so that we can have warning banners with features. 

# Why

Lets users know about the effects of the local camera switcher and whether it can be dragged with a warning: 
image

# How Tested

inspected docs locally to verify banner shows properly.

### CODE REVIEW

File: packages/storybook/stories/BetaBanners/StoryBookBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <Stack className={bannerContainerStyles(palette.background)}>
+      <Stack.Item>
+        {props.children && (
+          <Stack style={{ display: 'inline-block' }}>
+            <ImportantHeading color={palette.header} />
+            {' - '}
+            {props.children}
+          </Stack>
+        )}
+        {!props.children && (
```
Comment (JamesBurnside) [inline packages/storybook/stories/BetaBanners/StoryBookBanner.tsx:31]: This shouldn't pivot off prop.children, but be it's own component that uses StorybookBanner, i.e. something like
```
const SingleLineBetaBanner () => (
  <StoryBookBanner>
    <>
      <Text>This feature is currently in public preview and not recommended for production use. </Text>
      <Link  ... >
    </>
  </StorybookBanner>
);
```

File: packages/storybook/stories/BetaBanners/StoryBookBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * @private
+ */
+export interface StoryBookBannerProps {
+  palette?: BannerPalette;
+  children?: JSX.Element;
+}
+
+/** @private */
+export const StoryBookBanner = (props: StoryBookBannerProps): JSX.Element => {
```
Comment (JamesBurnside) [inline packages/storybook/stories/BetaBanners/StoryBookBanner.tsx:19]: nit: StoryBook -> Storybook (+props and filename)

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx [hunk 3]
```diff
@@ -129,6 +131,15 @@ const getDocs: () => JSX.Element = () => {
         button will cycle through the different camera's in the users device. Typical usage is to enable this button on
         mobile devices. This button is enabled through the use of the `showCameraSwitcherInLocalPreview` prop.
       </Description>
+      <StoryBookBanner
+        // eslint-disable-next-line react/no-children-prop
+        palette={yellowBannerPalette}
+      >
+        <p style={{ display: 'inline-block' }}>
```
Comment (JamesBurnside) [inline packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:138]: use fluent components over raw html elements
```suggestion
        <Text>
```

File: packages/storybook/stories/BetaBanners/StoryBookBanner.tsx (hunk 1 above)
Comment (dmceachernmsft) [reply packages/storybook/stories/BetaBanners/StoryBookBanner.tsx:31]: ah okay I see, will make this change!

**Review by JamesBurnside — APPROVED** [review]

**Comment by dmceachernmsft** [comment]
> image
> 
> The icons look like they are missing in the newer version

I believe this is from a issue when it was updating the snapshots, will run the jobs again. Nothing I wrote for this touches this..

**Comment by dmceachernmsft** [comment]
> > image
> > The icons look like they are missing in the newer version
> 
> I believe this is from a issue when it was updating the snapshots, will run the jobs again. Nothing I wrote for this touches this so nothing should have changed. I think we have seen this before in past PR's lately

yeah looks like is a CI thing, will watch the snapshots before I merge 
image


> > image
> > The icons look like they are missing in the newer version
> 
> I believe this is from a issue when it was updating the snapshots, will run the jobs again. Nothing I wrote for this touches this so nothing should have changed. I think we have seen this before in past PR's lately

yeah looks like this is a CI issue, will watch snapshots before I merge
image

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx [hunk 4]
```diff
@@ -129,6 +132,14 @@ const getDocs: () => JSX.Element = () => {
         button will cycle through the different camera's in the users device. Typical usage is to enable this button on
         mobile devices. This button is enabled through the use of the `showCameraSwitcherInLocalPreview` prop.
       </Description>
+      <StorybookBanner
+        // eslint-disable-next-line react/no-children-prop
```
Comment (prprabhu-ms) [inline packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:136]: Why do you need this? I don't see any explicit `children` prop here.

**Review by prprabhu-ms — APPROVED** [review]

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx (hunk 4 above)
Comment (dmceachernmsft) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:136]: oops I think I forgot to remove that!

Comment (dmceachernmsft) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx]: fixed!

**Review by alkwa-msft — APPROVED** [review]

**Pull request merged**